bulk-ingester/src/main/java/org/alfresco/hxi_connector/bulk_ingester/repository/InMemoryAlfrescoMetadataRepository.java

```diff
@@ -41,16 +41,38 @@
 class InMemoryAlfrescoMetadataRepository implements AlfrescoMetadataRepository
 {
     private final List<AlfrescoNode> nodes = new ArrayList<>();
+    private boolean primaryHierarchyEnabled;
```

Comment:
Is there a reason to use the Alfresco Connector with the primary hierarchy disabled? I couldn't think of one